ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
```

Comment:
I still need to wrap my head around the many node_modules directories but I'll say exit code 0 for a failed test looks like a bug somewhere.

This is what I see using inclusivedesign/centos7 box and various Node.js versions, outside containers:

# NODE 0.10.36

    [vagrant@sandbox-idi-centos7-node-10 universal]$ node -v
    v0.10.36
    [vagrant@sandbox-idi-centos7-node-10 universal]$ node tests/all-tests.js # Run tests

    module.js:340
        throw err;
              ^
    Error: Cannot find module 'infusion'
        at Function.Module._resolveFilename (module.js:338:15)
        at Function.Module._load (module.js:280:25)
        at Module.require (module.js:364:17)
        at require (module.js:380:17)
        at Object.<anonymous> (/home/vagrant/scm/GPII/universal/tests/all-tests.js:21:13)
        at Module._compile (module.js:456:26)
        at Object.Module._extensions..js (module.js:474:10)
        at Module.load (module.js:356:32)
        at Function.Module._load (module.js:312:12)
        at Function.Module.runMain (module.js:497:10)
    [vagrant@sandbox-idi-centos7-node-10 universal]$ echo $?
    8


# NODE 0.10.40

    [vagrant@sandbox-idi-centos7-node-10 universal]$ node -v
    v0.10.40
    [vagrant@sandbox-idi-centos7-node-10 universal]$ node tests/all-tests.js # Run tests

    module.js:340
        throw err;
              ^
    Error: Cannot find module 'infusion'
        at Function.Module._resolveFilename (module.js:338:15)
        at Function.Module._load (module.js:280:25)
        at Module.require (module.js:364:17)
        at require (module.js:380:17)
        at Object.<anonymous> (/home/vagrant/scm/GPII/universal/tests/all-tests.js:21:13)
        at Module._compile (module.js:456:26)
        at Object.Module._extensions..js (module.js:474:10)
        at Module.load (module.js:356:32)
        at Function.Module._load (module.js:312:12)
        at Function.Module.runMain (module.js:497:10)
    [vagrant@sandbox-idi-centos7-node-10 universal]$ echo $?
    8

# NODE 4.2.2

    [vagrant@sandbox-idi-centos7-node-4 universal]$ node -v
    v4.2.2
    [vagrant@sandbox-idi-centos7-node-4 universal]$ node tests/all-tests.js # Run tests
    module.js:339
        throw err;
        ^

    Error: Cannot find module 'infusion'
        at Function.Module._resolveFilename (module.js:337:15)
        at Function.Module._load (module.js:287:25)
        at Module.require (module.js:366:17)
        at require (module.js:385:17)
        at Object.<anonymous> (/home/vagrant/scm/GPII/universal/tests/all-tests.js:21:13)
        at Module._compile (module.js:435:26)
        at Object.Module._extensions..js (module.js:442:10)
        at Module.load (module.js:356:32)
        at Function.Module._load (module.js:311:12)
        at Function.Module.runMain (module.js:467:10)
    [vagrant@sandbox-idi-centos7-node-4 universal]$ echo $?
    1